# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gmp/endianess-cross.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gmp-4.1.4/configure.in.vanilla	2007-04-02 10:16:11.000000000 +0000
+++ gmp-4.1.4/configure.in	2007-04-02 10:16:18.000000000 +0000
@@ -1718,11 +1718,6 @@
 GMP_H_HAVE_FILE
 GMP_IMPL_H_IEEE_FLOATS
 
-# FIXME: Autoconf 2.52 AC_C_BIGENDIAN bombs when cross compiling, but in the
-# future the probes will be better and this override can be removed.
-if test "$cross_compiling" = yes; then
-  ac_cv_c_bigendian=unknown
-fi
 AC_C_BIGENDIAN
 # Enhancement: In the future AC_C_BIGENDIAN will accept these actions as
 # parameters.
